README.md

```diff
@@ -100,6 +96,11 @@ for sequence in sequences:
 >>> ATG TTC ATT CTT CCG GCA CTT ATC GAA
 ```
 
+### Diffusion Model
+A novel hierarchical transformer-based model that uses both Generative Pre-trained Transformers (GPT) (on individual gene sequences) and stable diffusion to capture the correct context and longer-range interactions in genome-scale datasets. This model enables us to prospectively model SARS-CoV-2 evolution by leveraging its generative capabilities.
```

Comment:
can you change it to
"A novel hierarchical language model with two levels: the top level uses a diffusion model to capture global context and longer-range interactions across the entire genome sequence; the bottom level uses a transformer LM for codon-level modeling, guided by the top-level diffusion model."